api/s2n.h

```diff
@@ -868,6 +868,17 @@ S2N_API extern int s2n_config_set_verify_after_sign(struct s2n_config *config, s
  */
 S2N_API extern int s2n_config_set_send_buffer_size(struct s2n_config *config, uint32_t size);
 
+/**
+ * Enable recieving of multiple TLS records in a single s2n_recv call
+ * 
+ * Legacy behavior is to return after reading a single TLS record which may not be the most
+ * efficient way to invoke this function, especially if larger receive buffers are used.
+ * 
+ * @param config The configuration object being updated
+ * @returns S2N_SUCCESS on success. S2N_FAILURE on failure
+ */
+S2N_API extern int s2n_config_enable_recv_multi_record(struct s2n_config *config);
```

Comment:
Do we want to make this more of a true/false setter? Maybe someday we'll change the default behavior and the legacy behavior will be opt-in

Reply:
sure, we can do that.